generators/app/templates/common/pyproject.toml

```diff
@@ -12,15 +12,14 @@ loguru = "^0.7.0"
 <% } -%>
 
 [tool.poetry.group.dev.dependencies]
-black = "^22"
 <% if (includeDvc) { -%>
 dvc = {extras = ["<%= dvcRemoteType %>"], version = "^2.58.1"}
 <% } -%>
 mypy = "^1.2"
 pre-commit = "^2"
 pytest = "^7"
 pytest-cov = "^3"
-ruff = "^0"
+ruff = "^0.1.3"
```

Comment:
Last version is 0.1.5

Reply:
done

Reply:
Should >= be used instead? https://iscinumpy.dev/post/bound-version-constraints/ advocates for this approach

Reply:
Very interesting, but I think we can handle this question later. I will read this article ASAP, and keep you updated!